101-hdinsight-linux-with-existing-default-storage-account/prereqs/.settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
The storageAccount should be able to be put into a separate resourceGroup - so remove this file and add an output/input for the resourceGroup the contains the storageAccount